examples/basic/variables.tf

```diff
@@ -31,3 +32,9 @@ variable "create_recovery_cluster" {
   type        = bool
   default     = false
 }
+
+variable "alternative_region" {
+  description = "Alternative region"
+  default     = "us-west-2"
```

Comment:
What happens when primary and secondary region are `us-west-2`?

Reply:
since we have to hardcode the example in main `us-east-1`, `us-west-2` this example will only work if those are the regions we build the app. ill hardcode them both instead of parameterizing